### ISSUE TITLE
[Translatable] Load translated null values

### ISSUE THREAD

**Issue by jarkt** [Atlantic18/DoctrineExtensions#1990]
I use Translatable with MongoDB.
In my application is a Symfony form and blank values are stored as null in the database.
It works also for translated fields. In this case the "content" field is missing in the database. Great.

But I can't load the document as expected. (As a workaround I have to ensure that not null but an empty string is saved.)

I got the following exception:

```
ErrorException:
Notice: Undefined index: content

  at vendor/gedmo/doctrine-extensions/lib/Gedmo/Translatable/TranslatableListener.php:477
  at Gedmo\Translatable\TranslatableListener->postLoad(object(LifecycleEventArgs))
     (vendor/symfony/doctrine-bridge/ContainerAwareEventManager.php:61)
  at Symfony\Bridge\Doctrine\ContainerAwareEventManager->dispatchEvent('postLoad', object(LifecycleEventArgs))
     (vendor/doctrine/mongodb-odm/lib/Doctrine/ODM/MongoDB/Hydrator/HydratorFactory.php:479)
  at Doctrine\ODM\MongoDB\Hydrator\HydratorFactory->hydrate(object(BlockExperience), array('id' => '5c64606d3f5151016d0fc852', 'header1' => 'x', 'text1' => 'x', 'image1' => array('original' => 'dev/jark/blockExperience/original_5c646190e5da98.86055208', 'preview' => 'dev/jark/blockExperience/preview_5c6461971403c6.26054268', 'stats' => array('geometry' => array('width' => 1924, 'height' => 1924), 'avgcolor' => array('r' => 188, 'g' => 197, 'b' => 197, 'a' => 1)), 'frontend' => 'dev/jark/blockExperience/frontend_5c6461986cbdb9.60568206'), 'linkText1' => 'link 1', 'route1' => object(RoutePage), 'header2' => 'y', 'text2' => 'y', 'image2' => array('original' => 'dev/jark/blockExperience/original_5c64619b3bc776.00059054', 'preview' => 'dev/jark/blockExperience/preview_5c6461a518d5d1.93806630', 'stats' => array('geometry' => array('width' => 3488, 'height' => 2400), 'avgcolor' => array('r' => 198, 'g' => 185, 'b' => 178, 'a' => 1)), 'frontend' => 'dev/jark/blockExperience/frontend_5c6461a6df5d12.33070576'), 'linkText2' => 'link 2', 'route2' => object(RoutePage)))
     (vendor/doctrine/mongodb-odm/lib/Doctrine/ODM/MongoDB/Persisters/DocumentPersister.php:476)
  at Doctrine\ODM\MongoDB\Persisters\DocumentPersister->refresh(object(MongoId), object(BlockExperience))
     (vendor/doctrine/mongodb-odm/lib/Doctrine/ODM/MongoDB/UnitOfWork.php:2067)
  at Doctrine\ODM\MongoDB\UnitOfWork->doRefresh(object(BlockExperience), array('00000000253d3d950000000066c4f4c7' => object(BlockExperience)))
     (vendor/doctrine/mongodb-odm/lib/Doctrine/ODM/MongoDB/UnitOfWork.php:2043)
  at Doctrine\ODM\MongoDB\UnitOfWork->refresh(object(BlockExperience))
     (vendor/doctrine/mongodb-odm/lib/Doctrine/ODM/MongoDB/DocumentManager.php:446)
  at Doctrine\ODM\MongoDB\DocumentManager->refresh(object(BlockExperience))
     (vendor/csmk-it/cosmik-bundle/src/Repository/Repository.php:246)
  at jarkt\CosmikBundle\Repository\Repository->translateOne(object(BlockExperience), 'de')
     (vendor/csmk-it/cosmik-bundle/src/Repository/Repository.php:116)
  at jarkt\CosmikBundle\Repository\Repository->getById('5c64606d3f5151016d0fc852')
     (vendor/csmk-it/cosmik-crud-bundle/src/Controller/CrudAddEditController.php:146)
  at jarkt\CosmikCrudBundle\Controller\CrudAddEditController->crudEdit(object(Request), 'blockExperience', '5c64606d3f5151016d0fc852', object(srcApp_KernelDevDebugContainer), object(TraceableValidator))
     (vendor/symfony/http-kernel/HttpKernel.php:150)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Request), 1)
     (vendor/symfony/http-kernel/HttpKernel.php:67)
  at Symfony\Component\HttpKernel\HttpKernel->handle(object(Request), 1, true)
     (vendor/symfony/http-kernel/Kernel.php:198)
  at Symfony\Component\HttpKernel\Kernel->handle(object(Request))
     (public/index.php:25)
```

**Comment by AkenRoberts** [comment]
Fixed in #1991.

**Comment by mente** [comment]
Hey guys,

The supposedly fix in #1991 modified existing behavior for us for missing translations. Now instead of falling back to default language translation is null. We're using MySQL and we do treat empty strings as missing translations.

Will try to come up with reproduce and potential fix which hopefully won't break behavior for @jarkt

**Comment by mente** [comment]
I'm sorry, the problem is happening since 2.4.25 for us when additional boolean flag `is_translated` was introduced, this last change is unrelated.
